playbook.yml

```diff
@@ -117,6 +117,7 @@
       monit_protection: true
       nginx_install_method: package
       nginx_default_enable: false
+      nginx_client_max_body_size: "15M"
```

Comment:
I dunno, [here](https://github.com/ANXS/nginx/blob/master/defaults/main.yml#L39) it's prefixed with `nginx_` and with this setup it worked on staging.

I recently checked our largest known form and it was 700KB. So any number north of that is sufficient. As for what's reasonable, 10 would certainly be a fair alternative. It's in the ballpark. But 100 leaves you susceptible to various forms of memory starvation attack, it's just too big if you're not careful about it. I don't have much preference as to what the number is so long as it's somewhere between 5M and 15M.